src/components/TodoAppComponent/NewTodo.tsx

```diff
@@ -0,0 +1,23 @@
+interface PropsNewTodo {
+  value: string,
+  setValue(val: string): void;
+}
+export const NewTodo = ({ value, setValue }: PropsNewTodo) => {
+  const createNewTodo = () => {
+
+  };
```

Comment:
why is this empty? 🤔

Reply:
Accidently starting doing next task here! According this part of the task adding function is unnecessary.